doc/schema/schema.md

```diff
@@ -40,10 +41,20 @@ Currently there are 2 beta implementations of this design:
  <li><a class="reference internal" href="#examples">Examples</a></li>
 </ol>
 
+## Terminology
+The terminology used in this specification is based on [xarray's terminology](http://xarray.pydata.org/en/stable/terminology.html), however, no xarray knowledge is assumed in this description. There are also some extensions particular to  the InferenceData case.
+
+* **Variable**: NetCDF-like variables are multidimensional labeled arrays representing a single quantity. Variables and their dimensions must be named. They can also have attributes describing it. Relevant terms related to InferenceData variables are: *variable_name*, *values* (its data), *dimensions*, *coordinates*, and *attributes*.
+* **Dimension**: The dimensions of an object are its named axes. A variable containing 3D data can have dimensions `[chain, draw, dim0]`, thus, its `0th`-dimension is `chain`, its `1st`-dimension is `draw` and so on. Every dimension present in an InferenceData variable must share names with a *coordinate*. Given that dimensions must be named, dimension and dimension name are used equivalents.
```

Comment:
Nit: is it required that each dimension correspond to a coordinate?  I think not in the sense that xarray won't raise an exception if this doesn't hold.  Suggestion: replace "must" with "should."

Reply:
xarray supports both dimensions without coordinates and coordinates without dimensions. I do not see any usecase for them in InferenceData/ArviZ (ArviZ does not support any of the two and I don't recall anybody complaining either) so even if xarray supports it, I would rather InferenceData "not supporting" it.